packages/gatsby-plugin-clerk/src/ssr/utils.ts

```diff
@@ -1,17 +1,29 @@
+import { ServerSideAuth } from '@clerk/types';
 import cookie from 'cookie';
 import type { GetServerDataProps } from 'gatsby';
 
 import type { AuthData } from './getAuthData';
 import { GetServerDataPropsWithAuth } from './types';
 
+interface ActJWTClaim {
```

Comment:
❓ Do we have this in our global types package? If not, maybe it should be put there as it seems like a code smell to have package-specific type declarations for the JWT claims that are universal.

Reply:
I've added a type for it [in another PR](https://github.com/clerkinc/javascript/pull/363/files#diff-1b3085fd39e1eda807520c318dd0f339c90c246db82fcb91f3b12d366a5e62bbR98), I guess I should include the change here as well. Fixing...